tutorials/autotvm/tune_relay_mobile_gpu.py

```diff
@@ -174,6 +174,11 @@ def get_network(name, batch_size):
 # This target host is used for cross compilation. You can query it by :code:`gcc -v` on your device.
 target_host = 'llvm -target=aarch64-linux-gnu'
 
+# Cross compiler which will be used to export library to shared object (.so file) for edge device
+# Edge device does not need to have build-essential package (gcc, g++, ld, etc) to be installed
+cc = 'aarch64-linux-gnu-g++'
```

Comment:
We need to add more follow up instructions on how to install such cross compiler as a part of note block, set the following flags to use cross gcc and default to False(because the on device compiler is still the easiest in many cases)

Reply:
- set `use_cross_compiler_to_export_lib=False` by default
- added instructions on how to install armv7/8 cross-compilers

Reply:
Please use the reStructured-text comment block 

- https://github.com/dmlc/tvm/blob/master/tutorials/cross_compilation_and_rpc.py#L34
- https://github.com/dmlc/tvm/blob/master/tutorials/cross_compilation_and_rpc.py#L111